SendLog.py

```diff
@@ -1668,6 +1669,43 @@ def run_scilab_func_request():
     return list_value
 
 
+# App route for getting scilab expression output for Expression Block
+
+@app.route('/getExpressionOutput', methods=['POST'])
+def run_scilab_func_expr_request():
+    (diagram, __) = get_diagram(get_request_id())
+    if diagram is None:
+        print('no diagram')
+        return
+
+    pathfortxtfile = diagram.sessiondir;
+    head = request.form['head']
+    exx = request.form['exx']
+    command = "exec('" + EXP_SCI_FUNC_WRITE + \
+    "');callFunctionAcctoMethod('" + pathfortxtfile + \
+    "','" + head +"','" + exx + "');"
+
+    try:
+        diagram.scilab_proc = run_scilab(command)
+    except FileNotFoundError:
+        return "scilab not found. Follow the installation instructions"
+
+    diagram.scilab_proc.communicate()
+
+    file_name = pathfortxtfile + "/" + "expr_set_value.txt"
+    # create a dictionary
+    exprs_value = {}
+    # create a array containing value for set parameters
+    var_array = ["ok","ok1","ipar","rpar","nz"]
```

Comment:
```suggestion
    var_array = ["ok", "ok1", "ipar", "rpar", "nz"]
```

After every comma, put a space